admin/services/prepare-check.service.js

```diff
@@ -80,14 +86,17 @@ function secondsBetweenNowAndPinExpiryTime (pinExpiry) {
 /**
  * Return an entity suitable for inserting into the `preparedCheck` table
  * @param {object} check
+ * @param {string} schoolTimezone
  * @return {object}
  */
-function constructPreparedCheck (check) {
+function constructPreparedCheck (check, schoolTimezone) {
+  const pinValidFromUtc = pinTimestampService.generatePinTimestamp(config.OverridePinValidFrom, startOfDay(), eightAmToday(), schoolTimezone)
```

Comment:
i would suggest `pinValidFromGenerator` or something more specific to its actual purpose